ddsketch/ddsketch.go

```diff
@@ -216,3 +217,57 @@ func (s *DDSketch) FromProto(pb *sketchpb.DDSketch) (*DDSketch, error) {
 		zeroCount:          pb.ZeroCount,
 	}, nil
 }
+
+// Encodes a DDSketch as bytes
+func (s *DDSketch) ToBytes() ([][]byte, error) {
```

Comment:
Is there a reason we aren't using `ToProto()` and then `proto.EncodeToBytes` ?  

I have a few concerns about this approach:
* we are creating and are responsible for maintaining a new serialization format
* The gob marshaller relies on fields staying named the same, which is easy for future devs to forget
* the gob marshaller is tied to go

Reply:
The reason is... I didn't think of it! But that's definitely better, we can just do that on our end without having to modify this library 🎉   I'll test it & then if that works I'll go ahead and close this ticket